Acceptance tests fail on verifying token update transaction

### Description

The mirrornode acceptance test `Validate Token Flow` failed at step `And the mirror node REST API should confirm token update`

```
2022-11-10T00:58:02.524-0600 INFO reactor-http-epoll-2 c.h.m.t.e.a.c.MirrorNodeClient GET http://35.239.255.55/api/v1/tokens/0.0.48850856: 200 OK 
2022-11-10T00:58:03.039-0600 INFO reactor-http-epoll-3 c.h.m.t.e.a.c.MirrorNodeClient GET http://35.239.255.55/api/v1/tokens/0.0.48850856: 200 OK 
2022-11-10T00:58:03.552-0600 INFO reactor-http-epoll-4 c.h.m.t.e.a.c.MirrorNodeClient GET http://35.239.255.55/api/v1/tokens/0.0.48850856: 200 OK 
2022-11-10T00:58:04.065-0600 INFO reactor-http-epoll-1 c.h.m.t.e.a.c.MirrorNodeClient GET http://35.239.255.55/api/v1/tokens/0.0.48850856: 200 OK 
2022-11-10T00:58:04.578-0600 INFO reactor-http-epoll-2 c.h.m.t.e.a.c.MirrorNodeClient GET http://35.239.255.55/api/v1/tokens/0.0.48850856: 200 OK 
2022-11-10T00:58:05.093-0600 INFO reactor-http-epoll-3 c.h.m.t.e.a.c.MirrorNodeClient GET http://35.239.255.55/api/v1/tokens/0.0.48850856: 200 OK 
2022-11-10T00:58:05.605-0600 INFO reactor-http-epoll-4 c.h.m.t.e.a.c.MirrorNodeClient GET http://35.239.255.55/api/v1/tokens/0.0.48850856: 200 OK 
2022-11-10T00:58:06.121-0600 INFO reactor-http-epoll-1 c.h.m.t.e.a.c.MirrorNodeClient GET http://35.239.255.55/api/v1/tokens/0.0.48850856: 200 OK 
2022-11-10T00:58:06.634-0600 INFO reactor-http-epoll-2 c.h.m.t.e.a.c.MirrorNodeClient GET http://35.239.255.55/api/v1/tokens/0.0.48850856: 200 OK 
  And the mirror node REST API should confirm token update                                                                                      # com.hedera.mirror.test.e2e.acceptance.steps.TokenFeature.verifyMirrorTokenUpdateFlow()
      java.lang.AssertionError: 
Expecting actual:
  "1668063434.082090003"
not to be equal to:
  "1668063434.082090003"
```

This is a side effect of reducing rest polling max attempts from 60 to 10 in this [PR](https://github.com/hashgraph/hedera-mirror-node/pull/4617)

### Steps to reproduce

Run the acceptance test scenario with a lower rest polling max attempts

### Additional context

_No response_

### Hedera network

testnet

### Version

v0.67.3

### Operating system

_No response_

--- a/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/RestPollingProperties.java
+++ b/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/RestPollingProperties.java
@@ -9,9 +9,9 @@ package com.hedera.mirror.test.e2e.acceptance.config;
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- * 
+ *
  *      http://www.apache.org/licenses/LICENSE-2.0
- * 
+ *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
@@ -44,7 +44,7 @@ public class RestPollingProperties {
 
     @Min(1)
     @Max(60)
-    private int maxAttempts = 10;
+    private int maxAttempts = 20;
 
     @NotNull
     @DurationMin(millis = 500L)

